python/plugins/processing/script/ScriptTemplate.py

```diff
@@ -44,7 +44,10 @@ class ExampleProcessingAlgorithm(QgsProcessingAlgorithm):
 
     def tr(self, string):
         """
-        Returns a translatable string with the self.tr() function.
+        Returns a string translated by the self.tr() function.
+
+        This function makes use of the QT5 ability to translate a given 
```

Comment:
```suggestion
        This function makes use of the QT5 ability to translate a given
```

```
./python/plugins/processing/script/ScriptTemplate.py:49:72: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:60:75: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:61:81: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:62:80: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:84:78: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:85:80: W291 trailing whitespace
./python/plugins/processing/script/ScriptTemplate.py:86:79: W291 trailing whitespace
```

and so on for other lines @tschmetzer

Reply:
> 
> 
> ```
> ./python/plugins/processing/script/ScriptTemplate.py:49:72: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:60:75: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:61:81: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:62:80: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:84:78: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:85:80: W291 trailing whitespace
> ./python/plugins/processing/script/ScriptTemplate.py:86:79: W291 trailing whitespace
> ```
> 
> and so on for other lines @tschmetzer.
> 
> You must remove whitespaces at the end of each lines.

Thanks, what a shame! I was mislead by the error message. I'll never forget what a trailing whitespace is :D